doc/classes/PopupMenu.xml

```diff
@@ -91,6 +107,8 @@
 			<argument index="3" name="global" type="bool" default="false">
 			</argument>
 			<description>
+				Adds an icon and assigns the specified [ShortCut] and [Texture] to it.
```

Comment:
You didn't explain what the global argument does. I just checked the source code quickly, I'm not sure what global shortcuts are either though.

Reply:
Sorry, I wasn't sure either.